docs/application/dotnet/tutorials/certificates/permit-device-to-install-apps.md

```diff
@@ -0,0 +1,14 @@
+# Permit Device To Install Applications
+
+In the menu bar, click `Tools > Tizen > Tizen Device Manager`.
```

Comment:
We don't use inline code for a menu name.

Reply:
What is inline code?

Reply:
Reference: https://guides.github.com/features/mastering-markdown/


Should be changed to:
```
In the menu bar, click **Tools** > **Tizen** > **Tizen Device Manager**.
```
or (if you don't want to use many emphasis)
```
In the menu bar, click **Tools > Tizen > Tizen Device Manager**.
```

Reply:
OK. Done.